### DISCUSSION TITLE
My school changed their email addresses

### DISCUSSION THREAD

**Discussion by not-nullptr** [community/community#43755] [GitHub Education] [answered]
Hello! I'm a current student and, as such, have been provided with an email address. This has been the standard since I joined the school in 2020. I wanted to apply for GitHub's student plan since I am a developer and would benefit from having GitHub Pro, but when I tried to apply it said I needed to have an email ending in the school's old domain name. GitHub Support directed me to here.

**Comment by maaicondgl** [comment, marked as answer]
vincula o email da  escola com seu github, acho que vai resolver seu problema.

**Reply by not-nullptr** [reply]
i've tried that, but the school's domain name has changed since they signed up to github's education program... would quite like a github employee to change the school's domain

**Reply by divais** [reply]
Can you please share the details so that we can update the school systems.

**Reply by not-nullptr** [reply]
@divais sorry for the late reply. is there any way i can tell you the details in private? i'd rather not post where i go to school on a public discussion board

**Reply by divais** [reply]
I understand  @madeline-xoxo, can you share your original support issue and we can take it forward there.

**Reply by not-nullptr** [reply]
@divais original issue as in ticket?

**Reply by divais** [reply]
yes please. Sorry for the inconvenience.

**Reply by not-nullptr** [reply]
@divais the ticket no. is 1945330. should we take this to some kind of dms?

**Comment by gustemet** [comment]
Same problem and github doesn`t help, just closed my ticket....

**Reply by cmumatt** [reply]
Ridiculous -- GitHub wouldn't accept my usual university e-mail and insisted on using one particular alias on one particular domain.

No part of this new system seems to actually work or make any sense in the real world.  Maybe it looked amazing on PowerPoint.

**Reply by not-nullptr** [reply]
@cmumatt 100% agreed. this is complete BS. i waited nearly 2 weeks for a response from support only to be redirected here by a canned response.

**Reply by cmumatt** [reply]
@madeline-xoxo Hard same.  I'll cross into week #3 of this hot mess tomorrow.  

I opened a support ticket, and GitHub auto-closed it today saying that starting today students need to support ourselves in this discussion forum. If GitHub's system isn't working, students can't fix that -- that's purely a GitHub issue.

**Reply by not-nullptr** [reply]
@cmumatt its like they waited until today to respond just so they didn't have to help us, lol

**Reply by cmumatt** [reply]
@madeline, sure seems that way. 

The ironic thing is I am head TA of a 120 student CS class starting next week, which uses GitHub classroom. 

This entire situation seems to be completely absurd.  I'm waiting for Bill Murray to personally hand-deliver a paper approval letter with a dozen roses because that would be the next absurd thing that might happen, lol.

**Reply by not-nullptr** [reply]
@cmumatt god that must suck, im just a student who wants the benefits. 120 students are gonna be missing out on github pro because of the poorly done support, that sucks ass

**Reply by divais** [reply]
@cmumatt Sorry for the inconvenience caused, can you please share your support issue #.

**Reply by gustemet** [reply]
My issue ticket no. #1943731 @divais

**Reply by samus-aran** [reply]
Hi @gustemet 👋 

Thank you again for your patience! We have updated the relevant information of the school you had identified. Please re-apply and let us know how you get on. 👍

**Reply by gustemet** [reply]
@samus-aran Thank you, now everything is okay!

**Comment by daniel-cues** [comment]
It makes no sense that Github support sent you here. Reopen the ticket or ask your school to contact Github

**Reply by not-nullptr** [reply]
a collaborator just fixed my issue and is about to fix @cmumatt's (who's having the same issue). if you're going to call me a liar then don't speak at all.

**Reply by daniel-cues** [reply]
Jeez you need to chill. Not everything is an attack. I was questioning Github's process, not your honesty. This isn't reddit SMH.
